nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/usermanager/GroupRootObject.java

```diff
@@ -75,13 +78,30 @@ private void checkGroupHasAName(NuxeoGroup group) {
     @Override
     boolean isAPowerUserEditableArtifact(NuxeoGroup artifact) {
         return isAPowerUserEditableGroup(artifact);
-
     }
 
     static boolean isAPowerUserEditableGroup(NuxeoGroup group) {
         UserManager um = Framework.getService(UserManager.class);
-        return !um.getAdministratorsGroups().contains(group.getName());
+        List<String> allGroups = computeAllGroups(um, group);
+        List<String> administratorsGroups = um.getAdministratorsGroups();
+        return allGroups.stream().noneMatch(administratorsGroups::contains);
+    }
+
+    protected static List<String> computeAllGroups(UserManager um, NuxeoGroup group) {
+        List<String> allGroups = new ArrayList<>();
+        computeAllGroups(um, allGroups, group);
+        return allGroups;
+    }
 
+    protected static void computeAllGroups(UserManager um, List<String> allGroups, NuxeoGroup group) {
+        allGroups.add(group.getName());
+        group.getParentGroups().stream().filter(pg -> !allGroups.contains(pg)).forEach(pg -> {
+            allGroups.add(pg);
+            NuxeoGroup parentGroup = um.getGroup(pg);
+            if (parentGroup != null) {
+                computeAllGroups(um, allGroups, parentGroup);
+            }
+        });
     }
 
```

Comment:
can be replaced by the same logic [here](https://github.com/nuxeo/nuxeo/pull/3620/commits/380d5003d35a6bb02a622d0f876109a51a361926#r360438691) to avoid recursive way to compute the groups ;)